private/data/i18n/en.json

```diff
@@ -696,9 +696,43 @@
       },
       "search": {
         "clearSearch": "Clear",
-        "noProductsFound": "No products found",
-        "searchInputLabel": "Search Reaction ",
-        "suggestedTags": "Suggested tags"
+        "noProductsFound": "No products found.",
+        "noOrdersFound": "No orders found.",
+        "noAccountsFound": "No accounts found.",
+        "searchInputLabel": "Search Reaction",
+        "suggestedTags": "Suggested tags",
+        "searchTypeProducts": "Products",
+        "searchTypeAccounts": "Accounts",
+        "searchTypeOrders": "Orders",
+        "accountSearchResults": {
+          "accountId": "Account ID",
+          "shopId": "Shop ID",
+          "firstName": "First Name",
+          "lastName": "Last Name",
+          "phone": "Phone",
+          "emails": "Email"
+        },
+        "orderSearchResults": {
```

Comment:
These address field declarations are probably also available in the addressBook and address? https://github.com/reactioncommerce/reaction/blob/7078918eaa969034f82a533a497dc73852c6579e/private/data/i18n/en.json#L546 - I'm going to suggest we keep these, as they are table column headers..